src/EventStore.Core.Tests/Common/ClusterNodeOptionsTests/when_building/with_cluster_node_and_custom_settings.cs

```diff
@@ -261,4 +268,94 @@ public class
 				_node.GossipAdvertiseInfo.InternalTcp);
 		}
 	}
+	
+	[TestFixture(typeof(LogFormat.V2), typeof(string))]
+	[TestFixture(typeof(LogFormat.V3), typeof(uint))]
+	public class with_cluster_custom_password_for_admin_and_ops_user<TLogFormat, TStreamId> : ClusterMemberScenario<TLogFormat, TStreamId> {
+		private const string _adminPassword = "Admin";
+		private const string _opsPassword = "Ops";
+
+		protected override ClusterVNodeOptions WithOptions(ClusterVNodeOptions options) =>
+			options with {
+				DefaultUser = new ClusterVNodeOptions.DefaultUserOptions { DefaultAdminPassword = _adminPassword, DefaultOpsPassword = _opsPassword }
+			};
+
+		[Test]
+		public void should_set_the_custom_admin_and_ops_user_password() {
+			Assert.AreEqual(_adminPassword,_options.DefaultUser.DefaultAdminPassword);
+			Assert.AreEqual(_opsPassword, _options.DefaultUser.DefaultOpsPassword);
+		}
+	}
+	
+	[TestFixture(typeof(LogFormat.V2), typeof(string))]
+	[TestFixture(typeof(LogFormat.V3), typeof(uint))]
+	public class with__cluster_custom_settings_check_for_environment_only_options<TLogFormat, TStreamId> : SingleNodeScenario<TLogFormat,
+			TStreamId> {
+		protected override ClusterVNodeOptions WithOptions(ClusterVNodeOptions options) =>
+			options with {
+			};
+		
+		private IConfigurationRoot _configurationRoot;
+
+		[Description("A flag."), DefaultValue(false)]
+		public bool Flag => _configurationRoot.GetValue<bool>(nameof(Flag));
+
+		[Description("An array of strings.")]
+		public string[] ArrayOfStrings =>
+			_configurationRoot.GetValue<string>(nameof(ArrayOfStrings))?.Split(',') ?? Array.Empty<string>();
+
+		[Description("An array of IP Endpoints.")]
+		public IPEndPoint[] IPEndPoints => _configurationRoot.GetValue<string>(nameof(IPEndPoints))?.Split(',')
+			.Select(IPEndPoint.Parse).ToArray() ?? Array.Empty<IPEndPoint>();
+
+		[Description("A sensitive value like a password."), Sensitive]
+		public string Sensitive => _configurationRoot.GetValue<string>(nameof(Sensitive));
+		
+
+		[Test]
+		public void should_return_error_when_default_password_options_pass_through_command_line() {
+
+			var args = new string[] {
+				"--DefaultAdminPassword=Admin2023#",
+				"--DefaultOpsPassword=Ops2023#"
+			};
+			_configurationRoot = new ConfigurationBuilder()
+				.Add(new DefaultSource(new Dictionary<string, object> {
+					[nameof(Flag)] = false,
```

Comment:
These configuration root options aren't used in these tests, so they could be removed and the default source set to just the options you're interested in, for example :

```
new Dictionary<string, object> {
	[nameof(ClusterVNodeOptions.DefaultUser.DefaultAdminPassword)] = "changeit",
	[nameof(ClusterVNodeOptions.DefaultUser.DefaultOpsPassword)] = "changeit",
}
```